code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -183,6 +217,7 @@
 
 	playsound(loc, 'sound/voice/liveagain.ogg', 75, 1)
 	AddMovementHandler(/datum/movement_handler/robot/use_power, /datum/movement_handler/mob/space)
+	RemoveMovementHandler(/datum/movement_handler/mob/physically_capable) // this lets robots move around when weakened. but they receive a speed debuff.
```

Comment:
![catsay](https://user-images.githubusercontent.com/61743710/160022400-dd6a3464-7779-4b75-a8e9-a8615c4a0445.gif)

Reply:
thanks for pointing out , forgot to remove it.